extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -71,14 +71,20 @@ import {upgradeBackgroundAudio} from './audio';
 const PAGE_LOADED_CLASS_NAME = 'i-amphtml-story-page-loaded';
 
 /**
- * Selectors for media elements
+ * Selectors for media elements.
+ * Only get the page media: direct children of amp-story-page (ie:
+ * background-audio), or descendant of amp-story-grid-layer. That excludes media
+ * contained in amp-story-page-attachment.
  * @enum {string}
  */
 const Selectors = {
   // which media to wait for on page layout.
-  ALL_AMP_MEDIA: 'amp-audio, amp-video, amp-img, amp-anim',
-  ALL_MEDIA: 'audio, video',
-  ALL_VIDEO: 'video',
+  ALL_AMP_MEDIA: 'amp-story-grid-layer amp-audio, ' +
```

Comment:
What about `background-audio` specified in an `amp-story`, do we want to also include that?

https://github.com/ampproject/amphtml/blob/a1ab9bd2678971e7e032f2b72fb5062fce79598d/extensions/amp-story/validator-amp-story.protoascii#L39

Reply:
This one is handled at the `amp-story.js` level, eg: https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/1.0/amp-story.js#L2042

Reply:
Nvm, just realized we don't do that normally, even with the `background-audio` of an `amp-story-page`.